plugins/patriot-aws/lib/patriot_aws/version.rb

```diff
@@ -1,4 +1,4 @@
 class VERSION
-  VERSION = "0.2.0.alpha"
+  VERSION = "0.3.0.alpha"
```

Comment:
please keep version as it is

Reply:
I modified version.
Could you check it again?